vcxagency-client/test/e2e/shared/agency-flows.spec.js

```diff
@@ -156,3 +139,29 @@ describe('healthchecks', () => {
     expect(success).toBe('true')
   })
 })
+
+describe('rate limiting', () => {
+  it('provision an agent, overload with messages, should fail', async () => {
```

Comment:
I would like to see test where only 1 agent crosses rate limit threshold, but the other one can still receive/download messages.